stripe-core/src/main/java/com/stripe/android/core/browser/customtabs/CustomTabsManager.kt

```diff
@@ -0,0 +1,186 @@
+package com.stripe.android.core.browser.customtabs
+
+import android.app.Activity
+import android.content.Context
+import android.net.Uri
+import androidx.annotation.RestrictTo
+import androidx.browser.customtabs.CustomTabsCallback
+import androidx.browser.customtabs.CustomTabsClient
+import androidx.browser.customtabs.CustomTabsIntent
+import androidx.browser.customtabs.CustomTabsServiceConnection
+import androidx.browser.customtabs.CustomTabsSession
+import androidx.lifecycle.DefaultLifecycleObserver
+import androidx.lifecycle.LifecycleOwner
+import com.stripe.android.core.Logger
+import javax.inject.Inject
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+interface CustomTabsManager : DefaultLifecycleObserver {
```

Comment:
We probably need to add a comment that this needs to be added as an observer to work

Reply:
https://github.com/stripe/stripe-android/pull/7081/commits/94176015bb8ce6b1e1783b72b3fba4b4547d9e7b